microsetta_interface/templates/source.jinja2

```diff
@@ -256,5 +274,16 @@
         {% endfor %}
     </div>
     {% endif %}
+    <hr style="border-top: dotted 1px"/>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-12 {% if samples|length > 0 %}tooltipper{% endif %}" data-title="Sources with one or more samples assigned cannot be deleted.  All samples above must be removed prior to deleting this source.">
+                <form name="delete_{{source_id}}_form" method="post" action="/accounts/{{ account_id }}/sources/{{ source_id }}/remove" onsubmit="return verifyDeleteSource('{{source_id}}');">
```

Comment:
I suggest passing `source_name` rather than `source_id` to the javascript confirm function; see comment above.

Reply:
Done